docs: update `Deploy application` article to guide the user to add a virtual host in next step (#965)
devopness/devopness: docs/applications/deploy-application.md

1. Review deploy information then click `Confirm`
   > A notification popup will be displayed, confirming that the deployment has been triggered
1. Wait for the `Application:Deploy` action to be completed

{% note %}

> **NOTE**: To make the application publicly accessible over the internet, follow the guide on how to {% mentionPost "/docs/virtual-hosts/add-virtual-host" %} in Devopness.

Reviewer: ```suggestion
**NOTE**: To make an application publicly accessible over the internet, follow the guide on how to {% mentionPost "/docs/virtual-hosts/add-virtual-host" %}
```
